recipes/libgd/all/conanfile.py

```diff
@@ -90,14 +101,18 @@ def generate(self):
         tc.variables["ENABLE_FONTCONFIG"] = False
         tc.variables["ENABLE_WEBP"] = self.options.with_webp
         if Version(self.version) >= "2.3.2":
-            tc.variables["ENABLE_HEIF"] = False
-            tc.variables["ENABLE_AVIF"] = False
+            tc.variables["ENABLE_HEIF"] = self.options.get_safe("with_heif", False)
+            tc.variables["ENABLE_AVIF"] = self.options.get_safe("with_avif", False)
         if Version(self.version) >= "2.3.0":
             tc.variables["ENABLE_RAQM"] = False
         tc.cache_variables["CMAKE_POLICY_DEFAULT_CMP0077"] = "NEW"
         tc.generate()
 
         deps = CMakeDeps(self)
+        deps.set_property("libheif", "cmake_file_name", "HEIF")
+        deps.set_property("WebP", "cmake_file_name", "WEBP")
```

Comment:
```suggestion
        deps.set_property("webp", "cmake_file_name", "WEBP")
```
Uhm, the first argument should be a Conan package name. I'm kind of surprised that it worked with a non-lowercase name.
Just fyi, it's worth keeping in mind that .set_property() does nothing if applied to a non-existent package or component.